src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -9,7 +16,22 @@ void EnSyatekiNiw_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnSyatekiNiw_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnSyatekiNiw_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_80B11DEC(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B132A8(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B131B8(EnSyatekiNiw* this, Vec3f* arg1, Vec3f* arg2, Vec3f* arg3, f32 arg4);
+void func_80B129EC(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B13464(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B123A8(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B11E78(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B12460(EnSyatekiNiw* this, GlobalContext* globalCtx);
+void func_80B128D8(EnSyatekiNiw* this, GlobalContext* globalCtx);
+s32 func_80B12FE0(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx);
```

Comment:
Probably should be `EnSyatekiNiw_OverrideLimbDraw`

Reply:
additionally, you shouldnt need to forward declare it

Reply:
I don't know how to change function names.

Reply:
literally just rename any occurrences of it